draft-ietf-core-coral.md

```diff
@@ -165,6 +167,60 @@ In contrast to RDF, however, the focus of CoRAL is not on the
 description of a graph of resources, but on the discovery of possible
 future application states.
 
+-->
+### Definitions
+
+The *CoRAL information model* is similar to the [Resource Description Framework (RDF)](#W3C.REC-rdf11-concepts-20140225) information model:
+Data is expressed as an (unordered) set of triples (also called statements),
+consisting of a subject, a predicate and an object.
+The predicate is always a URI in syntax normalized form, and subject and object are either a URI in syntax normalized form, or a concrete CBOR object (a literal).
+
+These triples form a directed graph with the subject and object being source and destination, and the predicate a description on the edge.
+That graph is equivalent to the data.
+
+To form a set and a graph, we define an equivalence relation:
+URIs are only equal to URIs and if they are identical byte-wise.
+Literals carry an identity from their serialization format (think of it as the file and cursor position from which it was read -- but if cbor-packed is involved it's more like the stack of cursors at parsing time),
+and are only equivalent to themselves.
```

Comment:
This para is a tad obscure.  Maybe what we want to say here that is simply that _"the value of a literal is its CBOR serialisation."_ ?